hoomd/md/integrate.py

```diff
@@ -121,25 +126,59 @@ class Integrator(_DynamicIntegrator):
     - `hoomd.md.methods.NPT`
     - `hoomd.md.methods.nph`
```

Comment:
Remove until ported.

Reply:
@b-butler I can remove `nph`, but isn't NPT already ported?

Reply:
Yes, but we didn't port `NPH` in Python.